index.html

```diff
@@ -192,8 +183,10 @@ <h2 class="about-head">About me</h2>
             <button class="btn" type="button">Get My Resume</button>
           </div>
         </div>
-        <hr class="about-separater" />
-        <div>
+        <div class="about1">
+          <hr class="about-separater" />
+        </div>
+        <div class="footer-technologies">
           <ul class="technologies">
             <li class="tech-title font-inter">
               <img src="./Skeleton/diamond.png" alt="diamond" />
```

Comment:
- [ ] Please try to align this section to be more in the center and spread across the view width, like in the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160218532-59613a7b-1939-4ff6-af4a-2b587f8f9d62.png) | ![image](https://user-images.githubusercontent.com/24703171/160218547-cb4e4af3-aa78-42bc-b2c3-b4b619efead9.png) |